packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -229,6 +238,40 @@ export interface SetLoanToken {
   interest?: BigNumber
 }
 
+export interface ListLoanTokenResult {
+  [key: string]: ListLoanDetail
+}
+
+export interface ListLoanDetail {
+  token: LoanTokenData
+  priceFeedId: string
+  interest: BigNumber
+}
+
+export interface LoanTokenData {
+  [key: string]: LoanTokenInfo
+}
+
+export interface LoanTokenInfo {
+  collateralAddress: string
+  creationHeight: BigNumber
+  creationTx: string
+  decimal: BigNumber
+  destructionHeight: BigNumber
+  destructionTx: string
+  finalized: false
```

Comment:
do we need to hardcode `false` to the interface?
```suggestion
  finalized: boolean
```

Reply:
Nope LOL